### ISSUE TITLE
Incorrect display of thumbnails in player

### ISSUE THREAD

**Issue by xeophyte** [UniversalMediaServer/UniversalMediaServer#4820]
I use VLC as player for UMS and for a long time in different places it shows incorrect thumbnails, like some pictures have thumbnails of videos from other localizations that were deleted or some photo/video thumbnails are repeated many times.
Reset the cache don't help. After resetting, after entering the folder with photos/videos, the thumbnails are reloaded and most of them are correct, but some photos/videos have mixed thumbnails.

UMS 14.2.0
VLC 3.5.4

**Comment by SubJunk** [comment]
@SurfaceS we have two reports of this in the last 2 weeks, any ideas?

**Comment by xeophyte** [comment]
Not fixed to me on 14.5. I cleared the UMS cache and disabled thumbnails, then restarted the UMS and VLC and some of the thumbnails were fixed, but others started to duplicate again in other places.

**Comment by SubJunk** [comment]
@xeophyte I think this is fixed, can you replicate it anymore?

**Comment by xeophyte** [comment]
Problem not fixed in 15.0 and newer VLC, and it is rather not VLC problem because I restored VLC to defaults and it exists. Is there any way to clear UMS thumbnail cache?
UMS 15.0
VLC 3.6.5
https://forum.videolan.org/viewtopic.php?f=35&t=164936

**Comment by Mik-S-UMS** [comment]
Try resetting the cache in UMS and this should create or download new thumbnails when your videos are scanned. This will keep your played progress.
<img width="475" height="215" alt="Image" src="https://github.com/user-attachments/assets/5c8a415d-f8bf-4961-84aa-ec42573c2f8f" />
Last time I used this UMS did not restart correctly so you may have to kill the javaw.exe processes in task manager if nothing happens after a few min and UMS is unresponsive.
I mentioned this [here](https://github.com/UniversalMediaServer/UniversalMediaServer/issues/5536#issuecomment-3401351471) so let me know if this happens to you too so this can be fixed if it is a problem.

**Comment by xeophyte** [comment]
I can't find this Navigation Settings. I don't use account, anonymous.

**Comment by Mik-S-UMS** [comment]
You don't need an account set up if you disabled it the first time running UMS, or if never set up it will ask you to create an admin account.
If you did set one up there is a way to reset this if you cannot remember the login details.

You should still be able to access the web settings by going to http://localhost:9001/settings

**Comment by xeophyte** [comment]
But I don't have this Navigation Settings, I think since v15.

<img width="489" height="216" alt="Image" src="https://github.com/user-attachments/assets/bd4b53a3-e0ed-4d3a-811d-27dac3b0854a" />

**Comment by Mik-S-UMS** [comment]
Check the show advanced options box in general settings and they will show.

<img width="287" height="194" alt="Image" src="https://github.com/user-attachments/assets/f96a84b1-9617-4b46-b5dd-bf2424f61483" />

**Comment by xeophyte** [comment]
Yes, resetting the cache didn't kill the javaw.exe (the 1st one).
Now, after clearing the cache and restarting, everything seems fine (no mixed thumbnails), but I'll keep an eye on it in the future to see if anything's broken.
The only new thing that's not working properly is that some videos (some old MP4 files in one of the folders) have random thumbnails from the internet.

![Image](https://github.com/user-attachments/assets/bed0740e-3f4d-4f7b-99f2-49a189ae71e3)
